diff --git a/node_modules/usb/binding.gyp b/node_modules/usb/binding.gyp
index 9f30d6b..8baa506 100644
--- a/node_modules/usb/binding.gyp
+++ b/node_modules/usb/binding.gyp
@@ -33,7 +33,7 @@
         'src/hotplug.cc'
       ],
       'cflags_cc': [
-        '-std=c++14'
+        '-std=c++17'
       ],
       'defines': [
         '_FILE_OFFSET_BITS=64',
@@ -61,7 +61,7 @@
           ['OS=="mac"', {
             'xcode_settings': {
               'OTHER_CFLAGS': [
-                '-std=c++1y',
+                '-std=c++17',
                 '-stdlib=libc++',
                 '-arch x86_64',
                 '-arch arm64'
